### PR TITLE
Update fr.m3u

### PR DESCRIPTION
**Pull request by dct-infra**
Updated TF1

### CODE REVIEW

File: channels/fr.m3u [hunk 1]
```diff
@@ -297,8 +297,8 @@ https://s6.tntendirect.com/rmcdecouverte/live/playlist.m3u8
 https://edge-live-ger1.ovstream.com/hls/10/index.m3u8
 #EXTINF:-1 tvg-id="TF1.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://i.imgur.com/CJAbJfM.png" group-title="Entertainment",TF1 (720p) [Offline]
 https://s6.tntendirect.com/tf1/live/playlist.m3u8
-#EXTINF:-1 tvg-id="TF1.fr" tvg-country="FR" tvg-language="French" tvg-logo="https://i.imgur.com/CJAbJfM.png" group-title="Entertainment",TF1 (302p) [Offline]
-http://stream.tvtap.net:8081/live/fr-tf1.stream/playlist.m3u8
```
Comment (Dum4G) [inline channels/fr.m3u:301]: Works for me. We should keep both variants

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Review by Dum4G — APPROVED** [review]

**Comment by Dum4G** [comment]
Removed an official link due to implemented token protection on https://www.tntendirect.com/TF1-en-direct @LaneSh4d0w

**Comment by LaneSh4d0w** [comment]
Thanks for tagging me, will take a look.

**Comment by vartaxe** [comment]
Tf1 seems to not work anymore

**Comment by dct-infra** [comment]
@vartaxe  works through the flusonic url

**Comment by vartaxe** [comment]
Oh sorry thanks ;)

**Pull request merged**